PhysicsTools/SelectorUtils/interface/VersionedIdProducer.h

```diff
@@ -86,7 +86,7 @@ VersionedIdProducer(const edm::ParameterSet& iConfig) {
     }
 
     if( idMD5 != calculated_md5 ) {
-      edm::LogError("IdConfigurationNotValidated")
+      edm::LogInfo("IdConfigurationNotValidated")
```

Comment:
Also, wouldn't say that a major feature is missing. It's possible to check git hashes as well (which appear to be more stable somehow...), so there are ways around it and it's not "a significantly missing piece and the validation is dead otherwise". One that is rather nice to have though, still possible to validate by git repo states.